src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -937,11 +937,18 @@ private void ScanNumericLiteralSingleInteger(ref bool underscoreInWrongPlace, re
         {
             if (TextWindow.PeekChar() == '_')
             {
-                underscoreInWrongPlace = true;
+                if (isHex || isBinary)
+                {
+                    // TODO(leading-digit-separator): check feature flag
```

Comment:
We don't let new TODOs into the master branch. 
Blockers should be addressed. Non-blockers should be tracked with a github issue.